$OpenBSD$

macro clashes (major/minor)
/usr/include/sys/types.h:#define        major(x) ...
/usr/include/sys/types.h:#define        minor(x) ...

--- plugin.c.orig	Tue Jul 10 15:47:04 2012
+++ plugin.c	Tue Jul 10 15:47:31 2012
@@ -162,12 +162,12 @@ zathura_plugin_manager_load(zathura_plugin_manager_t* 
       } else {
         girara_info("successfully loaded plugin %s", path);
 
-        zathura_plugin_version_t major = NULL, minor = NULL, rev = NULL;
-        g_module_symbol(handle, PLUGIN_VERSION_MAJOR_FUNCTION, (gpointer*) &major);
-        g_module_symbol(handle, PLUGIN_VERSION_MINOR_FUNCTION, (gpointer*) &minor);
+        zathura_plugin_version_t major1 = NULL, minor1 = NULL, rev = NULL;
+        g_module_symbol(handle, PLUGIN_VERSION_MAJOR_FUNCTION, (gpointer*) &major1);
+        g_module_symbol(handle, PLUGIN_VERSION_MINOR_FUNCTION, (gpointer*) &minor1);
         g_module_symbol(handle, PLUGIN_VERSION_REVISION_FUNCTION, (gpointer*) &rev);
-        if (major != NULL && minor != NULL && rev != NULL) {
-          girara_debug("plugin '%s': version %u.%u.%u", path, major(), minor(), rev());
+        if (major1 != NULL && minor1 != NULL && rev != NULL) {
+          girara_debug("plugin '%s': version %u.%u.%u", path, major1(), minor1(), rev());
         }
       }
 
